src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -91,7 +86,6 @@ function OptionRowLHNData({
     const reportID = propsToForward.reportID;
     // We only want to pass a boolean to the memoized component,
     // instead of a changing number (so we prevent unnecessary re-renders).
-    const isFocused = !shouldDisableFocusOptions && currentReportID === reportID;
```

Comment:
We can move this comment to `OptionRowLHNDataWithFocus`, since we moved the `isFocused` there as well.